layouts/index.html

```diff
@@ -33,7 +33,7 @@ <h1 class="bold-title {{ if or (.Site.Params.fadeIn | default true) .Site.Params
                     <!-- End top social icons -->
                 </div>
             </div> <!-- End of Hero body -->
-            {{ end }}
+            {{ end }}{{/* with */}}
```

Comment:
Would you please remove this? While I see the benefit, I would rather tackle this in a holistic change and discuss it in an issue before a PR.

Reply:
OK, I will. It was just that I was lost during the editing and this helped me to match the blocks.